pages.zh/linux/boltctl.md

```diff
@@ -0,0 +1,28 @@
+# boltctl
+
+> 控制雷电 (thunderbolt) 设备。
```

Comment:
Are you using halfwidth parens here on purpose? https://github.com/tldr-pages/tldr/pull/5240#issuecomment-775825707

Reply:
Yes, since the inside is all English.

Reply:
But what about @einverne's [comment](https://github.com/tldr-pages/tldr/pull/5240#issuecomment-775825707):

>According to this repository - [chinese-copywriting-guidelines](https://github.com/sparanoid/chinese-copywriting-guidelines), It is the recommended way to do it.

Reply:
Okay... I think it's better to just use this guideline for simplicity. It looks OK afterall.

I still strongly oppose the use of `《》` in this guideline, but we never encounter it in tldr anyways.